README.md

```diff
@@ -211,6 +212,10 @@ or
 make test.e2e.ocp
 ```
 
+## Release process
+
+Please refer to the [RELEASE-PROCESS.md](https://github.com/istio-ecosystem/sail-operator/blob/main/RELEASE-PROCESS.md) file for more information on how the Sail Operator release process works.  
+
```

Comment:
I would suggest to refer to the file within the repo and not by external link.
That way it would not be affected by any branch changes.

[RELEASE-PROCESS.md](RELEASE-PROCESS.md)

Reply:
Good point